pylint/config/arguments_manager.py

```diff
@@ -95,7 +95,20 @@ def __init__(self, prog: str, usage: str | None = None) -> None:
 
     @property
     def config(self) -> argparse.Namespace:
-        return self.namespace
+        """Namespace for all options."""
+        return self._config
+
+    @config.setter
+    def config(self, value: argparse.Namespace) -> None:
+        self._config = value
+
+    @property
+    def namespace(self) -> argparse.Namespace:
+        return self.config
+
+    @namespace.setter
+    def namespace(self, value: argparse.Namespace) -> None:
+        self.config = value
```

Comment:
```suggestion
```
Isn't it the moment where we change all our call to 'namespace' to 'config' ?

Reply:
Wanted to do that in a separate PR (due to the size of that change). Didn't want to let this important change go unnoticed!